### ISSUE TITLE
Parser fails on inline tables

### ISSUE THREAD

**Issue by lucab** [karupanerura/TOML-Parser#5]
I think this is strictly related to #2, but at the moment this is my biggest blocker.

The parser currently fails on inline tables. Even the example ones
```
name = { first = "Tom", last = "Preston-Werner" }
point = { x = 1, y = 2 }
```
fail with a
```
Syntax Error: line:1
name = { first = "Tom", last = "Preston-Werner" }
      ^
```

Can you please make the parser aware of inline tables? For your convenience, a large and updated corpus of TOML test sample is being maintained [here](https://github.com/BurntSushi/toml-test/tree/master/tests).

**Comment by karupanerura** [comment]
Thank you for reporting.
This module's current support version is v0.2.0. (https://github.com/toml-lang/toml/blob/master/versions/en/toml-v0.2.0.md#datetime)
I'll be support it on #2.
Please wait it. (or patches welcome)

**Comment by lucab** [comment]
Thanks for the quick answer!
Yes I guessed this was a version mismatch, but I reported anyway as it is the main blocker for a Debian packaging helper script.

Unfortunately I'm not an advanced enough perl-coder to touch a module internals.

**Comment by karupanerura** [comment]
oh!
I think your problem is very important.
OK. I preferentially start #2.
I can't promise, I'll implement the v0.4.0 in tomorrow. (but, I can't promise it.)

**Comment by lucab** [comment]
Don't worry, no hurry. I'll just keep tracking this.

**Comment by karupanerura** [comment]
Done. You can try on `feature/0.4` branch.

**Comment by karupanerura** [comment]
merged

**Comment by karupanerura** [comment]
Released it as version 0.6 to CPAN.

**Comment by lucab** [comment]
Thanks for the prompt reaction. This is so far working very well for all the TOML manifests I had to parse so far. I'll report back if I encounter further issues, but for the moment this works perfectly for me.
